dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -4387,7 +4396,8 @@ ReturnedColumn* buildFunctionColumn(Item_func* ifp, gp_walk_info& gwi, bool& non
 
     // A few functions use a different collation than that found in
     // the base ifp class
-    if (funcName == "locate" || funcName == "find_in_set" || funcName == "strcmp" || funcName == "regexp_instr")
+    if (funcName == "locate" || funcName == "find_in_set" || funcName == "strcmp" ||
```

Comment:
https://mariadb.com/kb/en/regexp_instr/ - REGEXP_INSTR(subject, pattern)

https://mariadb.com/kb/en/find_in_set/ - FIND_IN_SET(pattern, strlist)

These two have different position of "pattern" argument.

Reply:
@mariadb-SergeyZefirov I'm not quite understand a question. The change introduced in this patch for code line you commented is just a code formatting, which was applied by clang-format tool automatically based on clang-format config we have. 
In details: the expression `funcName == "regexp_instr"` was moved to line below, because we have limit for number of characters in one line.

Reply:
Sorry, I misread equality check as an assignment of the funcName.